src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/FileParser.cs

```diff
@@ -44,7 +44,8 @@ public IList<Rule> Parse(TextReader input)
                 switch (tokens[0])
                 {
                     case "RewriteBase":
-                        throw new NotImplementedException("RewriteBase is not implemented");
+                        throw new NotImplementedException("RewriteBase is not implemented because the notion of the " +
+                                                        "path base spans across all rules, not just mod_rewrite");
```

Comment:
In general, it is sufficient for the exception message to say simply "RewriteBase is not implemented". Explanation of why can go into a comment above the line next to the exception